dali/base/dafdesc.hpp

```diff
@@ -326,7 +328,8 @@ extern da_decl StringBuffer &makePhysicalPartName(
                                 unsigned replicateLevel,            // uses replication directory
                                 DFD_OS os,                          // os must be specified if no dir specified
                                 const char *diroverride,            // override default directory
-                                bool dirPerPart);                   // generate a subdirectory per part
+                                bool dirPerPart,                    // generate a subdirectory per part
+                                unsigned stripeNum);                 // strip number
```

Comment:
trivial: indent of comment and "strip"->"stripe"

Reply:
will fix in next commit